src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/ApplicationController.cs

```diff
@@ -113,7 +113,7 @@ public async Task<IActionResult> Create(string returnUrl = null)
                 _notifier.Warning(H["There are no registered services to provide roles."]);
             }
 
-            await foreach (var scope in _scopeManager.ListAsync())
+            await foreach (var scope in _scopeManager.ListAsync(null, null, default))
```

Comment:
Curious: why this change was required?

Reply:
Otherwise the tests failed. I could not Mock a ListAsync(). If you have a suggestion for the unittest, then I can adjust the unittest. It looks kind a strange with this parameters